src/server.js

```diff
@@ -355,9 +355,12 @@ const render = async (req, res, next) => {
       return;
     }
 
+    const pageTitle = route.title || 'Lunch';
+
     const data = { ...route,
       apikey: process.env.GOOGLE_CLIENT_APIKEY || '',
-      title: route.title || 'Lunch',
+      title: pageTitle,
+      ogTitle: req.path.includes('login') ? 'Lunch' : pageTitle,
```

Comment:
I don't like this case being here. Could you set `ogTitle` in the `login` route config instead?

Reply:
Sure